src/aof.c

```diff
@@ -2486,6 +2497,7 @@ int rewriteAppendOnlyFileRio(rio *aof) {
         }
         kvstoreIteratorRelease(kvs_it);
     }
+    serverLog(LL_NOTICE, "aofrw done, %llu keys skipped.", skipped);
```

Comment:
test？

Reply:
there is in the tests, please grep 'appendonly'

Reply:
i feel like `aofrw` is not the style in the redis log
```suggestion
    serverLog(LL_NOTICE, "AOF rewrite done, %llu keys skipped", skipped);
```

Reply:
and show total key count like bgsave log?

Reply:
it was copy paste from fork but I guess we can handle some diff here. I'll push the better log line.